src/util.cpp

```diff
@@ -788,13 +789,25 @@ const fs::path &GetDataDir(bool fNetSpecific)
         path /= BaseParams().DataDir();
 
     if (fs::create_directories(path)) {
-        // This is the first run, create wallets subdirectory too
-        fs::create_directories(path / "wallets");
+        // This is the first run, setup datadir
+        SetupDatadir(path);
     }
 
     return path;
 }
 
+void SetupDatadir(fs::path path)
+{
+    // create "wallets" subdirectory
```

Comment:
Nit: Redundant comment :-)  Next line says the same thing.

Reply:
I like the explicit comment here personally. It was there before this PR as part of a larger comment and I wanted to keep it around